client/json-rpc/src/async_client/tests.rs

```diff
@@ -36,18 +40,29 @@ macro_rules! assert_err {
 }
 
 fn setup(request: (&'static str, Value), response: Value) -> Client<Retry> {
-    setup_batch(vec![(request, response)])
+    setup_multi_requests(vec![(request, response)])
+}
+
+fn setup_multi_requests(inouts: Vec<((&'static str, Value), Value)>) -> Client<Retry> {
+    setup_with_server(
+        inouts
+            .iter()
+            .map(|((method, params), response)| {
+                (
+                    json!({"id": 1, "jsonrpc": "2.0", "method": method, "params": params}),
```

Comment:
the id here should be different for each request.

Reply:
each request is same because they are standalone request, not batch request in this func.